draft-ietf-quic-recovery.md

```diff
@@ -987,7 +988,7 @@ time_sent:
 : The time the packet was sent.
 
 
-## Constants of interest
+## Constants of interest {#constants-of-interest}
```

Comment:
```suggestion
## Constants of Interest {#constants-of-interest}
```

(I would have called this `{#constants}` instead; the tag you chose has no real effect.)